.evergreen/config/test/_template-local.yml

```
@@ -0,0 +1,56 @@
+tasks:
+  - name: "test-mongodb-%mongodbVersion%-standalone-noauth-nossl"
+    tags: ["standalone", "local", "%mongodbVersion%"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          TOPOLOGY: "server"
+          MONGODB_VERSION: "%mongodbVersion%"
+      - func: "run tests"
+  - name: "test-mongodb-%mongodbVersion%-standalone-auth"
```

```suggestion

  - name: "test-mongodb-%mongodbVersion%-standalone-auth"
```

Consider adding newlines between task names. It's done below, but inconsistently.

---

Added a number of newlines in templates and regenerated files. FWIW, I grouped tasks using the same topology together, but it's making the file harder to read. I'll consider using comments if we feel the need to group tasks.